resources/wikia/modules/flowTracking.js

```diff
@@ -0,0 +1,44 @@
+define('wikia.flowTracking', ['wikia.tracker', 'mw'], function (tracker, mw) {
+
```

Comment:
I'm wondering if we could add here some constants that define names of the flows